### PR TITLE
Mobile responsive design

### PR DESCRIPTION
**Pull request by EidHachem**
This PR includes the following:
1- Navigation bar and heading of portfolio
2- Mobile compatible design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+</head>
+<body>
+    <header>
+        <nav class="head">
+            <p class="name">Eid Hachem</p>
+           <div class="menu">
+               <div class="menu-icon icon1"></div>
+               <div class="menu-icon icon2"></div>
+               <div class="menu-icon icon3"></div>
+           </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly wrap the hamburger in the `nav tag`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <title>Eid Hachem</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <nav class="head">
+            <p class="name">Eid Hachem</p>
```
Comment (sja-thedude) [inline index.html]: - [x] Kindly wrap line 16 in `a tag` as its the logo. Fix this.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
+* {
+  background-color: #191765;
+}
+
+.head {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```
Comment (sja-thedude) [inline style.css:10]: - [x] Kindly add the background image (ellipse at the top left) as per the Figma design.
- Your design:
Screenshot 2022-03-17 at 11 44 16 AM

- Figma design:
Screenshot 2022-03-17 at 11 43 41 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @EidHachem, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good PR Title and Description 👍 
- All linters are passing 👍 

## Required
- [x] Kindly add your contact details to the readme file.
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (EidHachem) [reply index.html]: Hi @sja-thedude 
It is already in a nav tag with the logo, should I put it inside another nav tag?

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢

Hey @EidHachem, 

Good job implementing the changes requested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**